stashy/pullrequests.py

```diff
@@ -80,6 +80,19 @@ def can_merge(self):
             * there are conflicts that need to be manually resolved before merging; and/or
             * one or more merge checks have vetoed the merge.
         """
+        res = json.loads(self.merge_info())
```

Comment:
The merge_info() method returns a dict. I am running this commit with python 3.4 and getting 
```
    res = json.loads(self.merge_info())
  File "/usr/lib64/python3.4/json/__init__.py", line 312, in loads
    s.__class__.__name__))
TypeError: the JSON object must be str, not 'dict'
```